src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -188,435 +188,730 @@ If a problem occurs, check output in /var/log/proactive/scheduler
 
 WARNING: Once the upgrade is performed and the new version started for the first time, the ProActive server will upgrade <<_glossary_catalog,Catalog>> objects to the new version. This procedure ensures that any object modified manually by a user will not be upgraded.
 
-=== Install and Start ProActive using Docker Containers
+=== Automated ProActive Installation with Docker Compose
 ProActive could be easily deployed and started within a fully containerized environment. For this purpose, we provide a ready-to-use Docker Compose configuration that enables you running ProActive Workflows and Scheduling as a containerized service.
 It starts the following services:
 
-*  ProActive Server container
-*  ProActive Node container
-*  ProActive Database container (PostgreSQL or MySQL). The Database container is not started when using our embedded HSQL Database.
+*  ProActive server container
+*  (Optional) ProActive PostgreSQL database container. The database container is not started when using our embedded HSQL Database.
 
-The Docker Compose configuration pulls the ProActive server, Node, and Database images from the https://dockerhub.activeeon.com/[Private Activeeon Docker Registry^].
+The Docker Compose configuration pulls the ProActive server and database images from the https://dockerhub.activeeon.com/[Private Activeeon Docker Registry^].
 
-These images can be used out of the box, on any operating system. You can customize the server, Nodes, and Database settings, located in the `.env` file according to your requirements.
+These images can be used out of the box, on any operating system. You can customize the server and database settings, located in the `.env` file according to your requirements.
 
 To be able to access the https://dockerhub.activeeon.com/[Private Activeeon Docker Registry^], a ProActive enterprise licence and access credentials are required.
 Please contact us at contact@activeeon.com to request access.
 
 Here are the installation steps you need to follow in order to have ProActive running in a fully containerized environment:
 
-==== Configure the environment
+==== Download the Docker Compose Installation Package
+We provide a public Docker Compose installation package that you can use to configure and start the docker installation in your environment.
+It contains the Docker Compose configuration file (`.env`) and the YAML compose definitions (`docker-compose-hsqldb.yml` and `docker-compose-pg.yml`).
+You can proceed to donwload it using the following command:
 
-You need first to create and configure the Docker Compose environment file `.env`.
+'''
+++++
+<pre><code><span id="composeInstaller"/></code></pre>
+++++
+'''
```

Comment:
I don't see the download command

Reply:
You are right. This is because I am using JS to generate the link. It is not easy to understand but here is how the code is generated:  https://github.com/ow2-proactive/documentation/pull/907/files#diff-429ab7d50b78abcfc341af9b172b38bfd1b73e6fa3907ae439215346a59caf0cR5325

I agreed with @fviale to do so...